src/services/ApiInfo.tsx

```diff
@@ -0,0 +1,7 @@
+/* 
+ * The purpose of this class is to hold the default settings of OpenWeather API
```

Comment:
Are we still keeping the comments on this file? It is not adding much important information to the code readability, so it could be deleted.

btw, I'd rename this file to `AppConstants`, since this file is holding all important constants related to the API.